network-api/networkapi/buyersguide/templates/buyersguide_home.html

```diff
@@ -62,7 +62,7 @@ <h1 class="h1-heading text-center mt-4">Be Smart. Shop Safe.</h1>
           {% endif %}
         {% endif %}
 
-        <a class="product-image text-center mt-4 h-100 d-flex flex-column justify-content-between" href="/en/privacynotincluded/products/{{ product.slug }}">
+        <a class="product-image text-center mt-4 h-100 d-flex flex-column justify-content-between" href="privacynotincluded/products/{{ product.slug }}">
```

Comment:
We should keep the leading `/` 

i.e., `href="/privacynotincluded/products/{{ product.slug }}"`